Bottom: de3dfaf6cadf2cabbf418cc0797af3b535a34787
Top:    3e69dd8b87405c8918da8978e1412c95d681b1a8
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 11:16:34 +0800

Refresh of autofs-5.0.7-allow-use-of-hosts-map-in-maps.patch

---

diff --git a/modules/parse_sun.c b/modules/parse_sun.c
index 0dfd4f7..de09f2b 100644
--- a/modules/parse_sun.c
+++ b/modules/parse_sun.c
@@ -1294,8 +1294,8 @@ int parse_mount(struct autofs_point *ap, const char *name,
 	}
 
 	/*
-	 * If this is an indirect mount transform hosts map into a map
-	 * type so we don't confuse the options parsing.
+	 * Transform hosts map into a map type so we don't confuse the
+	 * options parsing.
 	 */
 	if (strstr(pmapent, "-hosts")) {
 		char *t = strstr(pmapent, "-hosts");
